src/lib/notion.ts

```diff
@@ -252,7 +253,7 @@ export const submitQuestion = async (param: SubmitQuestionArgs) => {
         title: [
           {
             type: 'text',
-            text: { content: generateNanoId() },
+            text: { content: nanoid() },
```

Comment:
This reverting previous changes. Is it intended?

Reply:
whoops, must be because i forget to pull last time